src/overlays/actors/ovl_En_Syateki_Wf/z_en_syateki_wf.h

```diff
@@ -19,19 +19,19 @@ typedef struct EnSyatekiWf {
     /* 0x20C */ Vec3s morphTable[WOLFOS_NORMAL_LIMB_MAX];
     /* 0x290 */ UNK_TYPE1 unk_290[0x4];
     /* 0x294 */ EnSyatekiWfActionFunc actionFunc;
-    /* 0x298 */ s16 unk_298;
-    /* 0x29A */ s16 unk_29A;
-    /* 0x29C */ s16 unk_29C;
-    /* 0x2A0 */ Vec3s* unk_2A0;
-    /* 0x2A4 */ s16 unk_2A4;
-    /* 0x2A6 */ s16 unk_2A6;
-    /* 0x2A8 */ s16 unk_2A8;
-    /* 0x2AC */ f32 unk_2AC;
+    /* 0x298 */ s16 isActive;
+    /* 0x29A */ s16 timer;
+    /* 0x29C */ s16 waitTimer;
+    /* 0x2A0 */ Vec3s* pathPoints;
+    /* 0x2A4 */ s16 currentPointIndex;
+    /* 0x2A6 */ s16 maxPointIndex;
+    /* 0x2A8 */ s16 yawTarget;
+    /* 0x2AC */ f32 unk_2AC; // set to 10.0f, but never used
     /* 0x2B0 */ u8 eyeIndex;
-    /* 0x2B4 */ ColliderCylinder unk_2B4;
-    /* 0x300 */ ColliderCylinder unk_300;
-    /* 0x34C */ ColliderJntSph unk_34C;
-    /* 0x36C */ ColliderJntSphElement unk_36C[1];
+    /* 0x2B4 */ ColliderCylinder bodyCollider;
+    /* 0x300 */ ColliderCylinder tailCollider;
+    /* 0x34C */ ColliderJntSph headCollider;
+    /* 0x36C */ ColliderJntSphElement headColliderElements[1];
     /* 0x3AC */ UNK_TYPE1 unk_3AC[0x20];
```

Comment:
Hmm, very mysterious, I don't suppose there's anything corresponding in the other shooting gallery actors? It doesn't fit with anything in Wf that I can see.

Reply:
It's almost certainly another `ColliderJntSph`, since it's exactly the right size for it. Other shooting gallery actors tend to all be similar *until* the colliders at the bottom, so I don't think we can look at them to figure out what this is. Instead, I think we have to wait until Wf is documented to guess at this; I'm assuming that Wf has a collider for one of its body parts that this was supposed to correspond to, but it was left half-implemented (there's no corresponding `ColliderJntSphElement` or `ColliderJntSphElementInit`) and unused.